src/js/tech/tech.js

```diff
@@ -10,10 +10,8 @@ import HTMLTrackElementList from '../tracks/html-track-element-list';
 import mergeOptions from '../utils/merge-options.js';
 import TextTrack from '../tracks/text-track';
 import TextTrackList from '../tracks/text-track-list';
-import VideoTrack from '../tracks/video-track';
```

Comment:
What about making that block of un-named imports in `player.js` the canonical place for these types of imports?